### PR TITLE
docs: Fix active color overflow of the Input with the search icon in t…

### PR DESCRIPTION
**Pull request by darknet-00**
Fix active color overflow of the Input with the search icon in the Search component

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

This is really a small fix. The problem was that when the input is active, the active color overflows (clashes) the search icon in the Search modal component. I will provide screenshots illustrating the before and after state of the UI. I couldn't find an open issue for this and I'm not sure If I need to open one before trying to contribute at all. If that's the case, I'll be happy to follow that rule.

---

## Screenshots

Current state: 
image

After the change:
image

PNPM check:
image

### CODE REVIEW

File: www/src/styles/algolia/style.css
Comment (c-ehrlich) [inline www/src/styles/algolia/style.css:66]:
```suggestion
  margin-inline-end: 15px;
```

This doesn't work in RTL languages as it places the margin on the wrong side. Could be fixed by using inline direction instead.

image

**Review by c-ehrlich — CHANGES_REQUESTED** [review]
Thanks for the PR, it's an improvement already. That being said now that this is being worked on I think it could also be improved on the "X" icon side and the bottom.

image

File: www/src/styles/algolia/style.css
Comment (darknet-00) [reply www/src/styles/algolia/style.css]: Thanks for the review. 

Oh yes, I didn't think about the RTL languages, thanks for your suggestion.

All of the mentioned issues should be fixed with the latest changes.

**Review by c-ehrlich — APPROVED** [review]
Looks great in both ltr and rtl, thanks :)

**Comment by c-ehrlich** [comment]
it seems like you have maintainer write access disabled in your fork so i can't merge the latest next branch into it

**Comment by darknet-00** [comment]
> it seems like you have maintainer write access disabled in your fork so i can't merge the latest next branch into it

It should be enabled now, I'm sorry about that.

**Pull request merged**
